lib/src/annotate.rs

```diff
@@ -0,0 +1,335 @@
+// Copyright 2020 The Jujutsu Authors
```

Comment:
2020?

Reply:
I copied from other files, wasn't sure if the year also gets updated

Reply:
This is a new file, so it should 2024

Reply:
should be resolved now